src/java/htsjdk/samtools/SAMTag.java

```diff
@@ -27,6 +27,53 @@
  * The standard tags for a SAM record that are defined in the SAM spec.
  */
 public enum SAMTag {
-    RG, LB, PU, PG, AS, SQ, MQ, NM, H0, H1, H2, UQ, PQ, NH, IH, HI, MD, CS, CQ, CM, R2, Q2, S2, CC, CP, SM, AM, MF,
-    E2, U2, OQ, FZ, SA, MC, BC
+    AM,
+    AS,
+    BC,
+    BQ,
+    CC,
+    CM,
+    CO,
+    CP,
+    CQ,
+    CS,
+    CT,
+    E2,
+    FI,
+    FS,
+    FZ,
+    GC, // for backwards compatibility
+    GS, // for backwards compatibility
+    GQ, // for backwards compatibility
+    LB,
+    H0,
+    H1,
+    H2,
+    HI,
+    IH,
+    MC,
+    MF, // for backwards compatibility
+    MD,
+    MQ,
+    NH,
+    NM,
+    OQ,
+    OP,
+    OC,
+    PG,
+    PQ,
+    PT,
+    PU,
+    QT,
+    Q2,
+    R2,
+    RG,
+    RT,
+    S2, // for backwards compatibility
+    SA,
+    SM,
+    SQ, // for backwards compatibility
+    TC,
+    U2,
+    UQ
 }
```

Comment:
Are these supposed to be in alphabetical order?
LB > H*
MD < MF
OC < OP, OQ
Letter+number should always go either before or after Letter+letter; it's a mix here